app.js

```diff
@@ -1,12 +1,26 @@
-let books = [];
+const books = JSON.parse(localStorage.getItem('books')) || [];
 
 const booksContainer = document.getElementById('books');
 const addBtn = document.getElementById('add-btn');
 const title = document.getElementById('title');
 const author = document.getElementById('author');
 
-if (JSON.parse(localStorage.getItem('books')) !== null) {
-  books = [...books, ...JSON.parse(localStorage.getItem('books'))];
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  addBook() {
+    const book = new Book(this.title, this.author);
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  removeBook() {
```

Comment:
- [ ] The **_removeBook()_** method is not working as intended. When the same book title and book author are entered twice or more and subsequently one of them is deleted, both all entries add with the same title and the author gets removed as well. We actually want each entry to have a **unique id**, or perhaps you can improve upon this aspect by **preventing duplicates**